### PR TITLE
Use svg logo in readme

### PR DESCRIPTION
**Pull request by kkocdko**
It is smaller (from `172 KB` to `8KB`) and fits the retina screen.

### CODE REVIEW

**Comment by fabiosantoscode** [comment]
Cool! We do want an SVG logo, let's see if it works well on NPM too. Next version, we'll know :)

**Comment by kkocdko** [comment]
@fabiosantoscode   I think I messed things up... `raw.githubusercontent.com` will not return a right meta for svg file. So now the logo can't be displayed :(

Maybe this logo should be placed on the [terser-js website](https://github.com/terser-js/terser-js.github.io). The GitHub Pages' server will return the right meta (I have tried before).

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 terser
 ======
 
-![Terser](https://raw.githubusercontent.com/terser-js/terser/master/logo.png)
+![Terser](https://raw.githubusercontent.com/terser-js/terser/master/logo.svg)
```
Comment (Martii) [inline README.md]: @kkocdko Cc: @fabiosantoscode

Re: https://github.com/terser-js/terser/pull/415#issuecomment-521357450

Needs additional QSP for GH of `?sanitize=true` appended to display in README.md. Tis a GH rule for showing SVGs. :)

Comment (Martii) [reply README.md]: Btw if you don't constrain it with a HTML `img` tag using  `width` and `height` it's going to fill the width and be rather large. You can of course size it  in the SVG itself *(usually CSS)* as well but that sort of defeats some of the purpose.

**Comment by fabiosantoscode** [comment]
Seems legit. Fixed height and width, and that GitHub querystring parameter.

New PR, or should I do these changes?

**Comment by Martii** [comment]
@fabiosantoscode 

I can do the PR for at least the GH QSP but it will be full width... and with xthexders Wide Github  userscript it's reallllllllllllllllly big. ;) It's been about six months since I dabbled with SVG CSS... but again... HTML constraining would probably be best so it's not "too small" on high DPI monitors. So I can submit the HTML code if you wish too... just need a baseline width to shoot for to calculate the correct height aspect ratio.

**Pull request merged**